### DISCUSSION TITLE
Get last publishedrevision of a collection

### DISCUSSION THREAD

**Discussion by Schloesslin** [directus/directus#16767] [Archive]
Hi !
I'm looking to get the last published revision of collection. The problem is that directus_revision.data is a JSON, so I can't filter on it.
Is there a solution to make that ?

**Comment by rijkvanzanten** [comment]
You can't currently search through json values yet, but one of our heroes @br41nslug is actively working on that functionality now! 🙂 

Make sure to keep an eye on https://github.com/directus/directus/pull/15889
